tljh_repo2docker/__init__.py

```diff
@@ -160,19 +166,24 @@ async def start(self, *args, **kwargs):
         return await super().start(*args, **kwargs)
 
 
-@hookimpl
-def tljh_custom_jupyterhub_config(c):
-    # hub
-    c.JupyterHub.hub_ip = public_ips()[0]
-    c.JupyterHub.cleanup_servers = False
-    c.JupyterHub.spawner_class = Repo2DockerSpawner
+if hookimpl:
```

Comment:
Maybe we should document why we have to conditionally define the TLJH hooks?

Probably to allow for the package to be used with the BinderHub chart on Kubernetes? If so, maybe the `tljh_repo2docker` should continue to focus on the TLJH integration (be a TLJH plugin) as before, and have a separate package for use with BinderHub.

Reply:
This could also likely be done separately, as the diff for this PR is already quite large.

Also the UI for building environments could eventually be its own `jupyterhub-environment-builder` package, and have `tljh_repo2docker` depend on it.

Reply:
Moved to https://github.com/plasmabio/tljh-repo2docker/issues/84